.changeset/dull-apples-film.md

```diff
@@ -0,0 +1,20 @@
+---
+"@nomiclabs/common": patch
+"@nomiclabs/hardhat-e2e-tests": patch
+"hardhat": patch
+"@nomiclabs/hardhat-docker": patch
+"@nomiclabs/hardhat-ethers": patch
+"@nomiclabs/hardhat-etherscan": patch
+"@nomiclabs/hardhat-ganache": patch
+"hardhat-shorthand": patch
+"@nomiclabs/hardhat-solhint": patch
+"@nomiclabs/hardhat-solpp": patch
+"@nomiclabs/hardhat-truffle4": patch
+"@nomiclabs/hardhat-truffle5": patch
+"@nomiclabs/hardhat-vyper": patch
+"@nomiclabs/hardhat-waffle": patch
+"@nomiclabs/hardhat-web3-legacy": patch
+"@nomiclabs/hardhat-web3": patch
+---
+
+update typescript and prettier versions
```

Comment:
I think it's better if we don't use a changeset here, as it's not a "releasable" change